### ISSUE TITLE
Feature Request: Apply Gain for Files/Stream w/o RG Tag

### ISSUE THREAD

**Issue by Quarkschleuder** [clementine-player/Clementine#5783]
It would be a good feature to be able to apply gain dependent on the existence of a ReplayGain Tag.
In my case I have my collection tagged properly but sometimes listen to a webradio stream or some file recently downloaded which is not tagged.

**Comment by nmw** [comment]
Yes.  And it's important to have the option to apply gain only to non-tagged files.  The way RG is typically implemented with a gain applied only to tagged files means that if you boost it 8dB to be in line with typical sources, any files with less than 8dB of gain tagged by RG will clip.  The correct behaviour is to apply a -8dB(or whatever value the user selects) to any files that don't have RG tags.
This way only music which RG tags with a positive gain are likely to clip.  Meanwhile any untagged files will be dropped to a more reasonable level.
